apps/token/src/i18n/translations/dev.json

```diff
@@ -625,5 +625,6 @@
   "FilterProposalsDescription": "Filter by proposal ID or proposer ID",
   "Freeform proposal": "Freeform proposal",
   "NewProposal": "New proposal",
-  "MinProposalRequirements": "You must have at least 1 VEGA to make a proposal"
+  "MinProposalRequirements": "You must have at least 1 VEGA to make a proposal",
```

Comment:
*question (blocking)*: It should be clearer what state this 1 vega needs to be in. And also it's not entirely true that it has to be 1 (see https://github.com/vegaprotocol/vega/issues/5786). But ignoring the number, could this be:

```suggestion
  "MinProposalRequirements": "You must have at least 1 VEGA staked to make a proposal",
```

It's also not quite true. It just has to be associated. But it's important because 1 VEGA in your general account won't count. @candida-d & me talked about this a lot on Friday so she'll understand where I'm coming from.